### PR TITLE
[RSDK-4137] Config changes for IMU Support

### PR DESCRIPTION
**Pull request by PStrutz**
- Add IMU config parameters in Go code.
- Add IMU into cartographer struct and cartofacade config
- Add IMU as an optional parameter in GetOptionalParameters
- Add tests for IMU config

### CODE REVIEW

File: cartofacade/capi.go
Comment (EshaMaharishi) [inline cartofacade/capi.go:327]: did you figure out what was missing here?

File: config/config.go
Comment (EshaMaharishi) [inline config/config.go:101]: instead of 'ok', let's say 'exists', since as a reader it's surprising to see an `if ok` without an `else` to handle the "not ok" case

File: config/config.go
Comment (EshaMaharishi) [inline config/config.go:114]: Ah, I see what you mean now about whether to leave it as the default rate or set it to 0. I think it can be left as the default rate.

**Review by EshaMaharishi — APPROVED** [review]
LGTM mod question about if you were able to get the code to auto-generate

File: cartofacade/capi.go
Comment (PStrutz) [reply cartofacade/capi.go:327]: Yep! Had to compile everything by running make build

**Comment by EshaMaharishi** [comment]
Ah you have a memory leak. We can talk about that and should fix it before merging

**Pull request merged**
